run_mount.go

```diff
@@ -51,6 +105,19 @@ func generateMountOpts(clicontext *cli.Context, imageVolumes map[string]struct{}
 			}
 			ociMounts[i] = x.Mount
 			mounted[filepath.Clean(x.Mount.Destination)] = struct{}{}
+
+			//copying up initial contents of the mount point directory
+			for imgVolRaw := range imageVolumes {
+				imgVol := filepath.Clean(imgVolRaw)
+				target := strings.Join([]string{tempDir, imgVol}, "")
```

Comment:
nit: why not `filepath.Join`?

Reply:
we should use securejoin. I'll push a commit